generator.js

```diff
@@ -0,0 +1,40 @@
+const fs = require('fs')
+const faker = require('faker')
+const argv = require('yargs').argv
+const helpers = require('./helpers/helpers.js')
+
+const residences = argv.residences || 10000000
+const filename = argv.output || 'residences.csv'
```

Comment:
So lines and 6 & 7 define our default values for these parameters in the event that you don't specify a value when invoking your script from the command line?